packages/lambdas/src/shared/cloudwatch.ts

```
@@ -74,7 +80,7 @@ export class CloudWatchUtils {
         .putMetricData({
           MetricData: [
             {
-              MetricName: "Memory total",
+              MetricName: metricName ?? "Memory total",
```

This could lead to confusion in my mind... likely you're not forcing it cause you don't want to update other code... but given we have prod alarms attached to metrics I'd expect us to throw if a metricName is undefined (or at least capture warning and not log).

---

Just flagging that a follow-up would be to set this explicitly if we're using it across different metrics... not defaulting to memory.

---

I see your point, but these are not related to alarms, but custom/domain metrics, under the `Metriport` namespace.

I'm adding a TSDoc to this function raising awareness about the fact `metricName` should almost always be defined.

---

done